.github/actions/python-environment/action.yml

```diff
@@ -45,7 +45,7 @@ runs:
     - name: Set up Poetry (${{ inputs.poetry-version }})
       shell: bash
       run: |
-        wget --tries=10 --waitretry 10 --retry-connrefused --https-only -O - https://install.python-poetry.org | POETRY_VERSION="${{ inputs.poetry-version }}" "$PYTHON_BINARY" -
+        POETRY_VERSION="${{ inputs.poetry-version }}" "$PYTHON_BINARY" "${{ github.action_path }}/ext/get_poetry.py"
```

Comment:
Need to use `github.action_path` here, check https://docs.github.com/en/actions/reference/workflows-and-actions/contexts